src/data/finset/basic.lean

```diff
@@ -775,10 +782,11 @@ instance : lattice (finset α) :=
   inf_le_right := λ s t a h, (mem_ndinter.1 h).2,
   ..finset.partial_order }
 
-/-! #### union -/
-
 @[simp] lemma sup_eq_union : ((⊔) : finset α → finset α → finset α) = (∪) := rfl
 @[simp] lemma inf_eq_inter : ((⊓) : finset α → finset α → finset α) = (∩) := rfl
+@[simp] lemma bot_eq_empty {α : Type*} : (⊥ : finset α) = ∅ := rfl
```

Comment:
Can you make a "Bounded order" section before "Lattice structure"? There you can avoid `decidable_eq α` on a section level.

Reply:
I'm not sure the section header is worth it for just two declarations, and this doesn't make things worse than they were before (which also had these two declarations in a section referring to decidability).

IMO it's useful to have all the "lattice vs set spelling" lemmas right next to each other.

Reply:
Eh sure